web-registry/src/components/organisms/ProjectMetadataForm/ProjectMetadataForm.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { Field, Form, Formik } from 'formik';
+
+import OnBoardingCard from 'web-components/lib/components/cards/OnBoardingCard';
+import ControlledTextField from 'web-components/lib/components/inputs/ControlledTextField';
+
+import ControlledFormLabel from 'web-components/lib/components/form/ControlledFormLabel';
+import { Body } from 'web-components/lib/components/typography';
+import { ShaclGraphByUriQuery } from '../../../generated/graphql';
+import { ProjectMetadataLD } from '../../../generated/json-ld';
+import { useProjectEditContext } from '../../../pages/ProjectEdit';
+import { ProjectPageFooter } from '../../molecules';
+import useProjectMetadataFormSubmit from './hooks/useProjectMetadataFormSubmit';
+import { validationSchema } from './ProjectMetadataForm.utils';
+
+interface ProjectMetadataFormProps {
+  submit: (values: ProjectMetadataValues) => Promise<void>;
+  initialValues?: ProjectMetadataLD;
+  graphData?: ShaclGraphByUriQuery;
+}
+
+export interface ProjectMetadataValues {
+  metadata: string;
+}
+
+const ProjectMetadataForm = ({
+  submit,
+  initialValues,
+}: ProjectMetadataFormProps): JSX.Element => {
+  const { confirmSave, isEdit } = useProjectEditContext();
+
+  const onSubmit = useProjectMetadataFormSubmit({
+    confirmSave,
+    isEdit,
+    submit,
+  });
+
+  return (
+    <Formik
+      enableReinitialize
+      validateOnMount
+      initialValues={{
+        metadata: initialValues ? JSON.stringify(initialValues) : '',
+      }}
+      validationSchema={validationSchema}
+      onSubmit={onSubmit}
+    >
+      {({ submitForm, isValid, isSubmitting, touched }) => {
+        return (
+          <Form translate="yes">
+            <OnBoardingCard>
+              <ControlledFormLabel>{'Project metadata'}</ControlledFormLabel>
+              <Body size="sm" mt={1} mb={3}>
+                {
+                  'Attach arbitrary JSON-LD metadata to the credit batch below. '
+                }
+              </Body>
```

Comment:
Do not use native `label` and `description` props from `Field` because we're supposed to have a link here later on.